src/i18next.js

```diff
@@ -1,31 +1,31 @@
-import i18n from 'i18next';

```

Comment:
also this one i do not know if it from the rebase of the develop or my bad 3gen work .